index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/index.css" />
+    <link
+      rel="stylesheet"
+      href="https://superui.netlify.app/stylesheets/reset.css"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://superui.netlify.app/stylesheets/color.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://superui.netlify.app/stylesheets/class.css"
+    />
+
+    <title>Quizophile</title>
+  </head>
+  <body>
+    <nav class="navigation flex--cemter">
+      <h1>Quizophile</h1>
+    </nav>
+
+    <div class="allQuiz">
+      <div class="quiz flex--cemter">
+        <img src="/Assets/Friends/friendsPNG.jpg" alt="" />
+        <div class="quiz--heading">Friends</div>
+        <div class="quiz--subHeading">
+          How well do you actually know friends .
+        </div>
+        <a href="/rules.css"><button class="btn">Play</button></a>
+      </div>
+
+      <div class="quiz flex--cemter">
+        <img src="/Assets/Squid Game/squidGamePNG.jpeg" alt="" />
+        <div class="quiz--heading">Squid Game</div>
+        <div class="quiz--subHeading">
+          If you win you win 456 Million dollars
+        </div>
+        <a href=""><button class="btn">Play</button></a>
```

Comment:
use # for blank href ,make it a dead link

Reply:
done